packages/shared/src/hooks/source/useSourceActionsNotify.ts

```diff
@@ -9,31 +9,32 @@ import { useNotificationPreferenceToggle } from '../notifications';
 import { useToastNotification } from '../useToastNotification';
 
 export type UseSourceSubscriptionProps = {
-  source: Pick<Source, 'id'>;
+  source: Pick<Source, 'id'> | Source;
 };
 
 export type UseSourceSubscription = {
-  isSubscribed: boolean;
+  haveNotifications: boolean;
   isReady: boolean;
-  onSubscribe: () => Promise<void>;
+  onNotify: () => Promise<void>;
 };
 
-export const useSourceSubscription = ({
```

Comment:
Is it really needed the name change, introduces a lot of changes to the code I think not really needed? Also its experiment which means if it fails we will revert this so better to keep changes minimal until we know its a win. Interested also to hear what others think.

Reply:
I've changed the name everywhere after [this slack thread](https://dailydotdev.slack.com/archives/C02E2C3C13R/p1721205082828849?thread_ts=1721204862.202729&cid=C02E2C3C13R)

Reply:
That was my logic yes 👍